helm/resource_release.go

```diff
@@ -88,6 +87,24 @@ func resourceRelease() *schema.Resource {
 					},
 				},
 			},
+			"sensitive_set": {
```

Comment:
To stay in the line of what we have already `set` `set_string` maybe this should be `set_sensitive`?

Reply:
Ah yes. I can get that changed.

Reply:
dome